[Bug] npx @yarnpkg/doctor triggers Internal Error: Cannot find module '@yarnpkg/fslib/sources'

- [ ] I'd be willing to implement a fix

**Describe the bug**

I am trying a migration & tried to run `npx @yarnpkg/doctor` which triggered an error.
Not sure if npx or yarn is the culprit.

```
❯ npx @yarnpkg/doctor
npx: installed 300 in 39.189s
Internal Error: Cannot find module '@yarnpkg/fslib/sources'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/moox/.npm/_npx/10733/lib/node_modules/@yarnpkg/doctor/node_modules/@yarnpkg/plugin-essentials/lib/commands/node.js:12:19)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

**To Reproduce**

I have no idea how to reproduce this since this might be something related to my setup. I can provide you as many things as you ask but I don't what to share in the first place except env info.

**Screenshots**

-

**Environment if relevant (please complete the following information):**

 - OS: macOS 10.15
 - Node version v10.15.1
 - Yarn version: via brew 1.22.4 / on my test project 2.0.0-rc.30

**Additional context**

I am on a react native project, using [fnm](https://github.com/Schniz/fnm) to handle my various node version. I am just trying to upgrade & see if that's usage because currently with yarn 1 I have issue to download a single package from a specific private repository (as .npmrc can only have a single registry line)

--- a/.yarn/versions/4c470e8b.yml
+++ b/.yarn/versions/4c470e8b.yml
@@ -1,0 +1,20 @@
+releases:
+  "@yarnpkg/cli": prerelease
+  "@yarnpkg/plugin-essentials": prerelease
+
+declined:
+  - "@yarnpkg/plugin-constraints"
+  - "@yarnpkg/plugin-dlx"
+  - "@yarnpkg/plugin-init"
+  - "@yarnpkg/plugin-interactive-tools"
+  - "@yarnpkg/plugin-node-modules"
+  - "@yarnpkg/plugin-npm-cli"
+  - "@yarnpkg/plugin-pack"
+  - "@yarnpkg/plugin-patch"
+  - "@yarnpkg/plugin-pnp"
+  - "@yarnpkg/plugin-stage"
+  - "@yarnpkg/plugin-typescript"
+  - "@yarnpkg/plugin-version"
+  - "@yarnpkg/plugin-workspace-tools"
+  - "@yarnpkg/core"
+  - "@yarnpkg/doctor"

--- a/packages/plugin-essentials/sources/commands/node.ts
+++ b/packages/plugin-essentials/sources/commands/node.ts
@@ -1,7 +1,7 @@
 import {BaseCommand}            from '@yarnpkg/cli';
 import {Configuration, Project} from '@yarnpkg/core';
 import {execUtils, scriptUtils} from '@yarnpkg/core';
-import {xfs}                    from '@yarnpkg/fslib/sources';
+import {xfs}                    from '@yarnpkg/fslib';
 import {Command, Usage}         from 'clipanion';
 
 // eslint-disable-next-line arca/no-default-export

